src/App.js

```diff
@@ -1,9 +1,52 @@
+import {useState,useEffect} from 'react';
+import Header from './components/Header';
+import BodyContainer from './components/BodyContainer';
+import members_data from './utils/data.json';
+import SearchBar from './components/SearchBar'
+import NoSearch from './components/NoSearch'
 
 function App() {
+  const [data, setData] =useState([]);
+  const [data1, setData1] =useState([]);
+  const [noSearch, setNoSearch] =useState(false);
+  const [text, setText] =useState('');
+  
+  useEffect(()=>{
+    getApiInfo();
+  },[])
+
+  const getApiInfo = async () => {
+  const allUsers = await getMemberInfo();
+    console.log(allUsers, "abcd")
+    setData(allUsers)
+  };
+
+  const getMemberInfo = async () =>{
+    return Promise.all(members_data.map(async (x) => {
+      return (await fetch(`https://api.github.com/users/${x.username}`)).json();
```

Comment:
API call is a reusable function so it can be defined & declared in a separate file to use it again somewhere else & also it will make App.js more clean.